server/server.go

```diff
@@ -35,6 +35,7 @@ import (
 	"github.com/pilosa/pilosa"
 	"github.com/pilosa/pilosa/gcnotify"
 	"github.com/pilosa/pilosa/gossip"
+	statik "github.com/pilosa/pilosa/statik/filesystem"
```

Comment:
would really like it if we could rename this package to avoid both stuttering and having to alias the import. I don't think I'm clear on why `filesystem` needs to be a subpackage of `pilosa/statik`.

Reply:
I combined those subpackages in the most recent commit. I thought I had tried combining them before but was blocked by a circular import, but I'm not sure what happened. Things should be organized pretty well now.